dev-support/docker/Dockerfile

```diff
@@ -84,6 +84,16 @@ RUN apt-get -q update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+#######
+# Boost 1.72 (1.65 ships with Bionic)
+#######
+WORKDIR /root/boost-library
```

Comment:
WORKDIR is `/root` in other places. Would you remove the overrides?

Reply:
Done @aajisaka. I've removed the WORKDIR overrides.